src/content/account/privacy-policy.md

```diff
@@ -200,6 +200,15 @@ The information provided in this “Notice to European Users” section applies
 
 **Controller.**  We are the controller of your personal information covered by this Privacy Policy for the purposes of European data protection legislation, except to the extent that we process your personal information on behalf of our customer, including personal information that we process on behalf of our Clients, in which case our customer is the controller of your personal information, and we are the processor.
 
+We have appointed a Data Protection Officer, whose contact information is:
+```
+DP Dock DPO Services GmbH
+Grüffkamp 10
+24159 Kiel
+Germany
+chromatic@dp-officer.com 
+```
+
```

Comment:
When you can, can you check the table below? It doesn't seem right to me. Specifically the 
**Processing Purpose**'s 2nd and 3rd rows

Reply:
Thanks, this is the correct format according to the internet